src/compiler.rs

```diff
@@ -121,17 +117,19 @@ fn CompileIdentifier(scope: usize, names: Option<&Vec<String>>, expr: Expression
 						} else {
 							panic!("This message should never appear");
 						};
-						checked += &format!("[({})]", rexpr);
+						write!(checked, "[({})]", rexpr).expect("");
```

Comment:
why do these write! calls have an empty string in the error message? While I really doubt it's ever gonna error maybe we should have it say "something really unexpected happpened" or something similar